microsite/data/plugins/kubelog.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Kubelog
+author: Julio Fernandez
+authorUrl: https://github.com/jfvilas
+category: Observability
+description: Kubelog provides your Backstage users the ability to view logs of your Kubernetes objects linked to Backstage entities. Security controls can be applied.
+documentation: https://github.com/jfvilas/kubelog
+iconUrl: https://raw.githubusercontent.com/jfvilas/kubelog/master/src/assets/kubelog-logo.png
+npmPackageName: '@jfvilas/plugin-kubelog'
```

Comment:
You package still points to the other account, here:

https://github.com/jfvilas/kubelog/blob/f462bd0f54c1cb07320916ca0c5dd4c4e9adab17/package.json#L24

And also here:

https://github.com/jfvilas/kubelog/blob/f462bd0f54c1cb07320916ca0c5dd4c4e9adab17/package.json#L32

Can you correct those please?